lib/l10n/intl_fr.arb

```diff
@@ -210,5 +210,14 @@
   },
   "go_to_dashboard": "Retour à l'accueil",
 
-  "no_connectivity": "Aucune connexion internet"
+  "no_connectivity": "Aucune connexion internet",
+
+  "progress_bar_title": "Progression de la session",
+  "progress_bar_message": "{elapsedDays} jours écoulés / {totalDays} jours",
```

Comment:
When no session is in progress, what does it show?

Reply:
Entre 2 sessions?

Reply:
par défaut il affiche 0

Reply:
Ouin genre entre la fin d'une session et la prochaine session. 

Donc ça affiche " 0 jours écoulés / 0 jours " ? 

Ça serait peut-être bien que ce soit écrit "Aucune session en cours" ou quelque chose du genre à la place ? Un peu comme schedule quand tu n'en a pas. Mais c'est juste une idée 😄

Reply:
Heum, il devrait toujours y avoir une session active, donc au départ il devrait afficher 0/107.

Reply:
Entre 2 sessions, la session suivante devient active, mais le nombre de jours complétés devrait rester à 0

Reply:
Ahhhh d'accord je comprends Parf alors ! :D

Reply:
Personnellement, je pense qu'afficher un message comme quoi la session n'est pas commencé pourrait être intéressant! Parce que exemple à Noël, le 0/107 serait afficher pendant 2 semaines et ça pourrait porter à confusion avec la première journée, mais c'est juste mon opinion hahaha. En même temps quand les étudiants sont en vacances, c'est rare que c'est une info qui est oublié de notre part 😉, donc ça me dérange pas trop.

Reply:
Voir autre commentaire ^^

Reply:
C'est possible de ne pas avoir de session actives, la session d'ete est une session optionnel, la majorite des étudiants partent en stage mais c'est possible que certains partent en vacances :smile:  c'est un cas a considérer tout comme l'entre deux sessions.